website/static/js/share/utils.js

```diff
@@ -1,4 +1,5 @@
 var m = require('mithril');
+var $osf = require('js/osfHelpers');
 var History = require('exports?History!history');
 
 var callbacks = [];
```

Comment:
the search is called in the searchBar module, so the stats module never has access to the results of the search. We just want to update the stats if the search is changed.